# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/greetd/tty-current.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Let our /etc/inittab decide which tty to run on.

  - NoTag <notag@t2sde.org>

--- greetd-0.9.0/config.toml.vanilla	2024-01-06 12:00:25.182732861 +0100
+++ greetd-0.9.0/config.toml	2024-01-06 12:00:36.464385005 +0100
@@ -1,7 +1,7 @@
 [terminal]
 # The VT to run the greeter on. Can be "next", "current" or a number
 # designating the VT.
-vt = 1
+vt = current
 
 # The default session, also known as the greeter.
 [default_session]
